www/jury/statistics.php

```diff
@@ -2,12 +2,9 @@
 
 require('init.php');
 
-// Include flot javascript library
 $extrahead = '';
-$extrahead .= '<!--[if lte IE 8]><script language="javascript" type="text/javascript" src="../js/flot/excanvas.js"></script><![endif]-->';
 $extrahead .= '<script language="javascript" type="text/javascript" src="../js/flot/jquery.js"></script>';
```

Comment:
Still including some flot component here, that doesn't seem right.

Reply:
Fixed.